### PR TITLE
[REF] mail: `FollowerSubtype`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2818653

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @seb-odoo @alexkuhn 👋
This PR is ready for review 👍

cc. @stefanorigano

File: addons/mail/static/src/components/follower_subtype/follower_subtype.scss [hunk 1]
```diff
@@ -1,27 +1,7 @@
...
-    padding: map-get($spacers, 2);
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
 .o_FollowerSubtype_label {
-    cursor: pointer;
+    flex-basis: 0;
```
Comment (alexkuhn) [inline addons/mail/static/src/components/follower_subtype/follower_subtype.scss:2]: Same remark as in `Follower` PR:
https://github.com/odoo/odoo/pull/88196#discussion_r846099802

I don't really get the reasoning of adding this rule.

**Review by alexkuhn — COMMENTED** [review]
LGTM otherwise, just pending for reply on reasoning about `flex-basis: 0;` rule addition.

File: addons/mail/static/src/components/follower_subtype/follower_subtype.scss (hunk 1 above)
Comment (Brieuc-brd) [reply addons/mail/static/src/components/follower_subtype/follower_subtype.scss]: Same answer 🙂  : https://github.com/odoo/odoo/pull/88196#discussion_r846124342

I removed it 👍

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**